### ISSUE TITLE
Communication server

### ISSUE THREAD

**Issue by JaqenGOT** [scakemyer/plugin.video.quasar#881]
Hello, first sorry if my english is not correct. Since I have a problem with quasar, it is impossible for me to load the providers, nothing is launched. I have an error when I run kodi with the communication server, I am told that it has a relationship with quasar and it is true that my problem has appeared since I have this message every time we start. Thank you for your help
M'y environement:
Android TV(MIBOX3)
Android version 6.0.1
Quasar version 0.9.78
Quasar Burst 0.5.10

**Comment by i3mYsElf** [comment]
I have  exactly the same issue, when this error appears it's impossible load any videos. Quasar burst dont load.

**Comment by tar-gezed** [comment]
Exactly same for me too on an Nvidia Shield TV

**Comment by JaqenGOT** [comment]
Then I found the solution is because of a "My Canal" application that causes uninstall because it conflicts with the communication service at the ports. It is an application to watch the French cable TV channels. For those who would have the same problem but in another country, look if you have not a similar application.

**Comment by tar-gezed** [comment]
I though of that during the night and it was the solution indeed. Thanks for sharing the solution :)

**Comment by i3mYsElf** [comment]
You have to uninstall it or just deactivate ?

**Comment by tar-gezed** [comment]
The MyCanal application isn't an addon, it's a real Android application, check if you haven't install a new app on your device that use the 9090 port ;)
You can do a `sudo lsof -i -P | grep -i "listen"` to view which application use a specific port on an UNIX system

**Comment by JaqenGOT** [comment]
You're welcome. Actually MY Channel is an Android application. To solve the solution we must thank rik91, with whom we shared to finally find the solution. It is in French: https://github.com/rik91/script.quasar.burst-provider/issues/3#issuecomment-323520502

**Comment by JaqenGOT** [comment]
Exactly, you should look into the android applications that are installed and that uses port 8080.

**Comment by JaqenGOT** [comment]
If you have a solution for this problem, I am a taker. Thank you
https://github.com/scakemyer/plugin.video.quasar
